wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController.swift

```diff
@@ -327,6 +333,54 @@ final class StartUIViewController: UIViewController {
         }
         return true
     }
+
+    private func presentCreateTeamBanner() {
+
+        typealias Localizable = L10n.Localizable.Peoplepicker
+        typealias Accessibility = L10n.Accessibility.Peoplepicker
+
+        let configuration = WireChannelBannerView.Configuration(
+            title: Localizable.UpgradeBanner.headline,
+            message: Localizable.UpgradeBanner.subheadline,
+            mainButtonTitle: Localizable.UpgradeBanner.Button.title,
+            mainButtonAction: { [weak self] in
+                self?.dismiss(animated: true) { [weak self] in self?.presentPersonalToTeamMigration() }
+            },
+            closeButton: .init(
+                accessibilityLabel: Accessibility.UpgradeBanner.CloseButton.label,
+                action: { [weak self] in self?.dismiss(animated: true) }
+            )
+        )
+        let banner = WireChannelBannerView(configuration: configuration)
+        // Dimmer that covers entire screen and intercepts taps
+        let rootView = ZStack {
+            Color.black.opacity(0.5)
+                .edgesIgnoringSafeArea(.all)
+            banner
+        }
+        .environment(\.wireTextStyleMapping, WireTextStyleMapping())
+
+        let hostingController = UIHostingController(rootView: rootView)
+        hostingController.view.backgroundColor = .clear
+        hostingController.modalPresentationStyle = .overFullScreen
+        hostingController.modalTransitionStyle   = .crossDissolve
+        hostingController.overrideUserInterfaceStyle = .dark
```

Comment:
It would be nice to specify in the banner itself that it should always appear in dark style.